### PR TITLE
First chapter translated to French

### PR DESCRIPTION
**Pull request by daimoc**
I start a French translation.

### CODE REVIEW

**Review by mogren — CHANGES_REQUESTED** [review]
Awesome, thank you! 

Just one thing, the "images" folder in content.fr should just be a soft link to the "docs/images/" one, no need to make duplicates. Also, there seem to be some broken pngs in there as well.

**Comment by daimoc** [comment]
Thank you for the review @mogren, I didn't see that the symlink was replaced by a real copy. 
I fixed it.

**Review by mogren — APPROVED** [review]

**Comment by mogren** [comment]
It looks good to me, but I'll give others a chance to comment before I submit it. Thanks again for the work, and if you find something in the english text that doesn't seem right, feel free to address it or open an issue about it. 😄

File: content.fr/_index.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+title: Introduction
+type: docs
+---
+
+# Le WebRTC pour les curieux
+
+Ce livre a été créé par des experts  WebRCT afin de partager leurs savoir durement acquis avec le monde.
```
Comment (hiroyuki-sato) [inline content.fr/_index.md:8]: WebRCT -> WebRTC (I can't read French by the way)
Thank you for your translation work!

Comment (daimoc) [reply content.fr/_index.md]: Thanks for catching this. It's fixed.

**Review by hiroyuki-sato — APPROVED** [review]

**Comment by mogren** [comment]
C'est magnifique!  https://webrtcforthecurious.com/fr/

**Pull request merged**